source/scss/modules/_report.scss

```diff
@@ -1,6 +1,6 @@
 .report {
   font-family: $secondary-font;
-  font-size: 0.75em;
+  font-size: 1.5rem;
```

Comment:
I wonder if there is another work around to pass this contrast error aside from enlarging the text to this degree. At this size I think it takes away from the sparkbox logo a bit and is yelling at the user to report a problem 😂. A size 1rem looks like it could be a happy medium? It also seems to pass the contrast ratio test.

<img width="447" alt="Screen Shot 2022-02-03 at 1 30 38 PM" src="https://user-images.githubusercontent.com/34315878/152406897-821f758b-71a1-4cc3-ae13-68686bd46c8c.png">

Reply:
Sorry, I lied! It looks like 1rem passes the browsers contrast test but not the contrast test in Lighthouse. Another alternative could be to keep the font size at `0.75` and update the font color to the `$c--iceberg` in the color config or we can also add a more subtle color like `#88b3bd` which is a lighter version of the `$c--juniper` thats there currently. These are a little more subtle but still pass the Lighthouse contrast requirement.

Reply:
Lol, I did think the same thing about the font size. I also didn't see the second comment until just now. But I ended up creating a new color variable that was able to pass and added it the all the content in the footer. Let me know what you think and if there are any other changes that could be made!

Reply:
Great idea! I think the color is a little bright, I think our safest bet is trying to keep the color as close to the original as possible but just light enough to still pass the contrast requirement. I think the closest color to junpier that still passes the contrast is `#82a7ae`? See how that works and we can plug it into the junpier-light variable you created.

Reply:
I think that color looks great, just made the change to that variable!